### ISSUE TITLE
Split into separate spacemacs specific package and generic packages

### ISSUE THREAD

**Issue by alexmurray** [paperwm/gnome-shell-mode#28]
To enable gnome-shell-mode to be used in standard vanilla emacs, it would be great to have the spacemacs parts split out into a separate package OR to at least document and support using this as a standard emacs package without spacemacs layers etc.

**Comment by olejorgenb** [comment]
Yes! That have been on the todo list a long time, but since both me and @hedning use spacemacs it has never happened :)

I think it should work to require the packages in 'local', but you'll need to set up keybindings yourself (and probably configure company )

Any advice on how to restructure is welcome (I'm no emacs expert)

**Comment by alexmurray** [comment]
ok no worries - will have a look and provide some suggestions.
